src/main/java/uk/gov/companieshouse/company/profile/service/CompanyProfileService.java

```diff
@@ -33,4 +34,18 @@ public CompanyProfileService(Logger logger, CompanyProfileRepository companyProf
                 companyNumber, companyProfileDao));
         return companyProfileDao;
     }
+
+    /**
+     * Updated insolvency links in company profile.
+     * @param companyProfileRequest company Profile information {@link CompanyProfile}
+     */
+    public void update(final CompanyProfile companyProfileRequest) {
+        CompanyProfileDao companyProfile = companyProfileRepository
+                .findByCompanyNumber(companyProfileRequest.getData().getCompanyNumber());
+        String insolvencyLink = companyProfileRequest.getData().getLinks().getInsolvency();
+        companyProfile.companyProfile.getData().getLinks().setInsolvency(insolvencyLink);
+        companyProfileRepository.save(companyProfile);
+        logger.debug(String.format("Data saved in company_profile collection : %s",
+                companyProfileRequest.toString()));
```

Comment:
We've established a convention around log statements to help testers easily find their way around in the logs:
`logger.trace(String.format("DSND-xyz: Data saved in company profile collection: %s", loggedItemHere))`